AutoNumber/CreateAutoNumber.cs

```diff
@@ -39,81 +38,81 @@ public class CreateAutoNumber : CeledonPlugin
 		// Registration Details:
 		// Message: Create
 		// Primary Entity: cel_autonumber
-		// User Context: SYSTEM
+		// User context: SYSTEM
 		// Event Pipeline: Post
 		// Mode: Async
 		// Config: none
 		//
 
-		internal const string PLUGIN_NAME = "CeledonPartners.AutoNumber.{0}";
+		internal const string PluginName = "CeledonPartners.AutoNumber.{0}";
 
 		public CreateAutoNumber()
 		{
-			RegisterEvent(POSTOPERATION, CREATEMESSAGE, "cel_autonumber", Execute);
+			RegisterEvent(PipelineStage.PostOperation, PipelineMessage.Create, "cel_autonumber", Execute);
 		}
 
-		protected void Execute(LocalPluginContext Context)
+		protected void Execute(LocalPluginContext context)
 		{
-			Trace("Get Target record");
-			Entity Target = Context.GetInputParameters<CreateInputParameters>().Target;
-			string pluginName = String.Format(PLUGIN_NAME, Target.GetAttributeValue<string>("cel_entityname"));
+		    context.Trace("Get Target record");
```

Comment:
not an immediate request but might pull these trace calls

Reply:
Will put this in the backlog :)